dataactbroker/handlers/account_handler.py

```diff
@@ -295,11 +308,11 @@ def perms_to_affiliations(perms, user_id):
                 continue
 
         perm_level = perm_level.lower()
-        if perm_level not in 'rwsefa':
+        if perm_level not in 'rwsef':
```

Comment:
This will log a warning every time for `a` permissions, which isn't what we want. We either want to switch this content and the content under the `elif` below, or completely separate the `elif` below into its own `if` statement above.

Reply:
swapped the logic to avoid the unnecessary warnings